javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+const arr = [
+  {
+    isbn: '9780375828379',
+    title: 'Hop On Pop',
+    author: 'Dr.Seuss',
+  },
+  {
+    isbn: '0679735771',
+    title: 'American Psycho',
+    author: 'Bret Easton Ellis',
+  },
+  {
+    isbn: '0393355942',
+    title: 'Fight Club: A Novel',
+    author: 'Chuck Palahniuk',
+  },
+];
+
+console.log('array typeof: ', typeof arr);
+
+const stringify = JSON.stringify(arr);
+console.log(stringify);
+console.log('JSON type of: ', typeof stringify);
+
+const newString =
+  '[{"Number id": "12345", "name":"Thomas"}, {"Number id":"56789", "name":"Shawn"}]';
```

Comment:
Property names should always start with lowercase, and shouldn't include spaces in them. You should change your property name so that it's just `id`